### ISSUE TITLE
[02/26] today I read

### ISSUE THREAD

**Issue by nmhkahn** [nmhkahn/paper-notes#4]
## Controllable Text-to-Image Generation
- Controllable 한 text-to-image 네트워크: 텍스트와 연관된 영역만 변경되는 등의 fine한 control이 필요함
- 기존에 사용되던 spatial attention 뿐만 아니라 word를 key, query로 하는 channel-wise attention을 같이 사용함 <- 특정 channel에서 특정 word에 대응되는 fine한 attention을 생성함
- Word-level discriminator: 텍스트에 연관된 영역만 변경되도록 fine하게 가이드
: discriminator에 unet base를 사용하면 더 fine하게 가이드를 줄수있을듯

## ManiGAN: Text-Guided Image Manipulation
- ControlGAN을 베이스로 text-driven image manipulation
- ACM: SPADE와 비슷한 모듈인데, segmap 대신 image에서 뽑은 regional feature, hidden feature는 텍스트에서 뽑은 feature
    - h*W(v)는 텍스트에 따라 어떤 위치를 변경할지를 정하고, b(v)는 text-irrelvent한 영역을 주로 나타냄
- DCM: 마지막 G에 넣는 input을 좀 더 fine하게 refine
- 학습할 때 manipulation용 (I, S) -> S` triplet이 없기 때문에 (I, S) -> I 방식으로 학습함; 따라서 b(v)가 identity를 배울수 있는듯
    - Reg loss를 넣어서 diversity를 보장

## MirrorGAN: Learning Text-to-image Generation by Redescription
- Text -> image -> text로 redescription한 뒤에 text recon loss 적용
- Image -> text module (STREAM)은 pretrain 해놓고 fix
- Text -> image module은 AttnGAN과 거의 유사한듯

## Learn, Imagine and Create: Text-to-Image Generation from Prior Knowledge
- Text-to-image task를 1) learn, 2) imagine, 3) create 단계로 분할
- Learn 단계는 image, text, segmask를 통해 image-text feature가 같은 공간에 있도록 맵핑, 이 단계는 pre-train됨
- Imagine 단계는 feature refine 및 noise add 단계
- Create 단계는 attnGAN이랑 비슷하게 생성하는듯

## YOU ONLY NEED ADVERSARIAL SUPERVISION FOR SEMANTIC IMAGE SYNTHESIS 
- Segmentation-based D: D가 per-pixel segmentation 하도록 설계 + fake 이미지인 경우 seg label 대신 fake label을 찍도록 (N+1) classes.
- LabelMix: Semantic 단위로 mask를 미리 만들어놓고 D(LabelMix(x, x_hat)) == LabelMix(D(x), D(x_hat))이 되도록 reg을 붙여줌. UNet-based D에서 비슷하게 가져온듯

## TOWARDS FASTER AND STABILIZED GAN TRAINING FOR HIGH-FIDELITY FEW-SHOT IMAGE SYNTHESIS 
- Skimming 만 했는데, self-supervised D (auto-encoding) 이 중요해보임
- OASIS는 D에서 seg+real/fake를 생성하는 점에서 비슷한 아이디어?
